### ISSUE TITLE
Ensure consistency of selectorNode over whole deloyment process

### ISSUE THREAD

**Issue by lukas-bednar** [kubevirt-incubator/kubevirt-ansible#7]
In order to deploy KubeVirt pods on master node, we need to set selectorNode in manifests.
Right now the playbook is taking FQDN of master node and use it as selector.

On other hand openshift-ansible allows you to change name of node in inventory file, and if this happen the assumption above will be false.

So we need to make sure that we have proper label assigned to node or take that `openshift_host` variable under consideration.

In additional I would change line in inventory file
https://github.com/kubevirt-incubator/kubevirt-ansible/blob/34e790e827f130909b652c9ef76130f08e237113/inventory#L22

Since it is required to have master node to be schedulable .
Without that the KubeVirt pods will not be be able to run.

copied from https://github.com/cynepco3hahue/kubevirt-ansible/issues/8

**Comment by fabiand** [comment]
Another way to solve this is to get rid of the cluster specific details.

The work of @stu-gott to use api server aggregation shoudl do a large part here.
The remaining gap is to expose the proxy and fix [virt-api](https://github.com/kubevirt/kubevirt/issues/457).

The proxy exposure can then be done using ingress resources, which are cluster dependent.

With these changes we effectively change the manifest to only define the inner cluster setup, which should be cluster agnostic. The remaining bits are to expose certain parts of the cluster to the outside world, and this can be handled separetly.

Long story short, I don't see the need for doing something with host fqdns or nodeSelectors.

**Comment by rmohr** [comment]
Hi! It is not a requirement to have all kubevirt components on the master node. That was just for developing /testing.

**Comment by gbenhaim** [comment]
@lukas-bednar still relevant?
